src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -323,7 +323,8 @@ def run_sessions_query(
             {column for field in fields for column in field.get_groupby(query.raw_groupby)}
         ),
         orderby=orderby,
-        limit=Limit(max_groups),
+        limit=Limit(min(query.limit or max_groups, max_groups)),
+        offset=Offset(query.offset or 0),
```

Comment:
When the API client sends no pagination parameters, `query.limit` is 51 at this point, while actually we need to return "all" the results ("all" in quotes because we always have the `max_groups` limit). This would be a breaking change to the public API. Maybe we can solve this by setting `default_page_size` of the paginator to `SNUBA_LIMIT`.

Reply:
Did you find out yesterday how we solved this in metrics_v2?

Reply:
I think it's still broken there, and only solved for the non-orderby case: https://github.com/getsentry/sentry/pull/31918. But I think raising the default `per_page` is a better solution.